pom.xml

```diff
@@ -1544,7 +1563,7 @@
     <properties>
         <!-- Carbon platform version comes here-->
 
-        <carbon.kernel.version>4.4.14</carbon.kernel.version>
+        <carbon.kernel.version>4.4.37-SNAPSHOT</carbon.kernel.version>
```

Comment:
Shall we upgrade to kernel released version

Reply:
This branch (master_java11) is related to giving JDK 11 support for the products. So still the kernel hasn't been released with JDK 11 support so we have used this snapshot version